### ISSUE TITLE
Daily details: add content description to tabs

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#1642]
Text in tabs read as follow: Sun, 23, Feb.
We should have a content description that allows it to be read by screenreaders as: "Sunday, February 23th".
